### ISSUE TITLE
Cannot load from file 0.6.25 RC1

### ISSUE THREAD

**Issue by coladict** [Fenoxo/sourceTiTS#541]
I get about 20 lines of `Error #1069`. This applies to the version as of the current HEAD of the master branch when submitting.

**Comment by Gedan** [comment]
Pls to be providing save file in question.

**Comment by coladict** [comment]
Both saves are from official android build 0.6.23. They load fine in the released desktop version, but have load errors in RC1 on both desktop and mobile.
[saves.zip](https://github.com/Fenoxo/sourceTiTS/files/76410/saves.zip)

**Comment by Gedan** [comment]
I figured out what this was; tl;dr, some of the serialization code I wrote tries to grab certain get/set function values. Given I like to occasionally use read-only properties from time to time, I slapped in some stuff to allow me to manage a list of property names to ignore. For some reason the ignore worked fine for regular fields, but not for accessors. Basically, it was saving a property to file that it couldn't then set when it came time to load, so it'd fall on its face and die about it.
